modules/identity/src/Volo.Abp.Identity.Application.Contracts/Volo/Abp/Identity/IIdentityRoleAppService.cs

```diff
@@ -8,6 +8,8 @@ namespace Volo.Abp.Identity
 {
     public interface IIdentityRoleAppService : IApplicationService
     {
+        Task<List<IdentityRoleDto>> GetAllListAsync();
```

Comment:
Please use ListResultDto wtihh has Items collection. We use this "wrapping" to be able to add new properties to the object without breaking existing client applications (backward compability).

Reply:
`ListResultDto` has been used.